changedetectionio/processors/text_json_diff.py

```diff
@@ -164,7 +164,8 @@ def run(self, uuid, skip_when_checksum_same=True, preferred_proxy=None):
             is_html = False
             is_json = False
 
-        if watch.is_pdf or 'application/pdf' in fetcher.get_all_headers().get('content-type', '').lower():
+        is_pdf = watch.is_pdf or 'application/pdf' in fetcher.get_all_headers().get('content-type', '').lower()
```

Comment:
Are there any other types like `is_pdf`?
Implicit sequence may break your feature by this PR.